api/handler/v1beta1/appeal.go

```diff
@@ -195,6 +195,33 @@ func (s *GRPCServer) RevokeAppeal(ctx context.Context, req *guardianv1beta1.Revo
 	}, nil
 }
 
+func (s *GRPCServer) RevokeAppeals(ctx context.Context, req *guardianv1beta1.RevokeAppealsRequest) (*guardianv1beta1.RevokeAppealsResponse, error) {
+	filters := &domain.ListAppealsFilter{}
+	filters.AccountIDs = req.GetAccountIds()
+	filters.ProviderTypes = req.GetProviderTypes()
+	filters.ProviderURNs = req.GetProviderUrns()
+	filters.ResourceTypes = req.GetResourceTypes()
+	filters.ResourceURNs = req.GetResourceUrns()
+
+	reason := req.GetReason()
+
+	appeals, err := s.appealService.BulkRevoke(ctx, filters, domain.SystemActorName, reason)
+	if err != nil {
+		return nil, status.Error(codes.Internal, "failed to revoke appeals in bulk")
+	}
```

Comment:
Could just pass the reason if it is not being used anywhere
```suggestion
	appeals, err := s.appealService.BulkRevoke(ctx, filters, domain.SystemActorName, req.GetReason())
	if err != nil {
		return nil, status.Error(codes.Internal, "failed to revoke appeals in bulk")
	}
```